content/core-features/channels.textile

```diff
@@ -0,0 +1,31 @@
+---
+title: Channels
+section: core-features
+index: 23
+---
+
+Ably aggregates all it's data into named units of distribution, referred to as "channels". Channels offer a way to implement the Publish-Subscribe (Pub/Sub) architectural pattern, which is a popular pattern used for realtime data delivery. 
+
+The Publish-Subscribe messaging pattern lets any number of publishers publish data to a channel, which could be subscribed to by any number of subscribers. The key thing to note about Pub/Sub is that publishers and subscribers are completely decoupled as explained further. Besides, once subscribed, the subscribers no longer have to poll the server or data provider to check if there is any new data that they need to be aware of; instead, they will be notified of it as it becomes available.
```

Comment:
`as explained further` => `as explained further down this page` or something to that effect, basically indicate the context of `further`. Also, link to where it's explained here?

Reply:
`Besides, once subscribed...` => `Once subscribed,...`